### PR TITLE
fix(ui): assortment of ui improvements

### LINKED ISSUE #282: Update Buttons internal alignment
* [x] Arrow should be same height as typography

### PR DESCRIPTION
**Pull request by carlos-kryha**
closes #282 #279 #278 #277 

### Description
Bundled a few ui bugs into this one:

> General
 - [x] Arrow should be same height as typography
 - [x] Make buttons hovering render a pointer instead of normal cursor
> About page
 - [x] Add link to Obscura word 
 - [x] Remove paragraph about about credits: "While playing ... every 24hs..." on about page
> How to play page
- [x] Change "Results Invalidated" for "0 Points"
- [x] Leaderboard section
- [x] Scores in each round make up a players total score
- [x] Ranks are based on total scores
- [x] Players Profiles section:
- [x] Anon players are ineligible for leaderboard positions
- [x] Profiles are created by connecting your wallet

### CODE REVIEW

**Review by ramidecodes — APPROVED** [review]

File: src/assets/text/about.ts [hunk 1]
```diff
@@ -5,8 +5,7 @@ export const about = {
...
     `Enter the BotBuster Program. It's a game that feeds questions into a chatroom. The kind of questions that humans like you and I can answer, but still trip up those nasty AI.`,
     `Stay sharp. Each conversation is a clever dance of wits and gut feelings, separating genuine human interactions from sly AI tricks. These bots will say anything to convince you they're the real deal.`,
-    `BotBusters is built by Kryha Labs on the zero-knowledge network Aleo using real privacy-preserving technology with zPass that can prove who's a real human without revealing their secrets to anyone. We can't even see it.`,
-    `While playing, you can earn points based on how well you do. Every day, the top BotBusters players with the most points will receive Aleo Credits, and this bounty resets every 24 hours.`,
+    `BotBusters is built by Kryha Labs, using Obscura, on the on the zero-knowledge network Aleo using real privacy-preserving technology with zPass that can prove who's a real human without revealing their secrets to anyone. We can't even see it.`,
```
Comment (silimarius) [inline src/assets/text/about.ts:8]:
```suggestion
    `BotBusters is built by Kryha Labs, using Obscura, on the zero-knowledge network Aleo using real privacy-preserving technology with zPass that can prove who's a real human without revealing their secrets to anyone. Not even we can see them.`,
```

File: src/assets/text/how-to-play.ts [hunk 2]
```diff
@@ -46,22 +45,21 @@ export const howToPlay = {
...
-    "Aleo credits are awarded to top 10 players automatically when the leaderboard resets.",
-    "Leaderboard resets daily at 00:00 PST.",
+  leaderboard: [
+    "Ranks are based on total scores.",
+    "Scores in each round make up a players total score",
   ],
   playerProfiles: [
     {
-      regular: "Anon players are ineligible for leaderboard/credits.",
+      regular: "Anon players are ineligible for leaderboard position.",
```
Comment (silimarius) [inline src/assets/text/how-to-play.ts:62]:
```suggestion
      regular: "Anonymous players are ineligible for leaderboard position.",
```
idk "Anon" doesn't seem straightforward if you are not a designer or developer

File: src/components/top-ranked/styles.ts [hunk 3]
```diff
@@ -1,4 +1,6 @@
 import { type SxProps } from "@mui/material";
+import { theme } from "~/styles";
+import { zIndex } from "~/styles/z-index";
```
Comment (silimarius) [inline src/components/top-ranked/styles.ts:3]: Extensions

**Review by silimarius — APPROVED** [review]

File: src/assets/text/how-to-play.ts (hunk 2 above)
Comment (carlos-kryha) [reply src/assets/text/how-to-play.ts:62]: too close to Qanon 😬

**Pull request merged**